src/main.ts

```diff
@@ -31,8 +31,11 @@ Vue.use(Vue2Filters) // needed by SbcFeeSummary
 const store = getVuexStore()
 const pinia = getPiniaStore()
 
-// Needed to fix "getActivePinia was called with no active Pinia" error
-// Type assertion to turn off type checking
+// attach global Vuex store to Vue instance
+Vue.prototype.$store = store
```

Comment:
This is key, since you get the global Vuex store only the first time you call getVuexStore() and we need to store its handle somewhere.